tests/dragonfly/pymemcached_test.py

```diff
@@ -12,119 +13,6 @@
 DEFAULT_ARGS = {"memcached_port": 11212, "proactor_threads": 4}
 
 
-# Generic basic tests
```

Comment:
there are no  changes in this file besides moving most tests under the same class to use the same process